Changelog.md

```diff
@@ -1,21 +1,23 @@
-### 0.4.24 (unreleased)
+### 0.4.24 (2018-05-16)
 
-Features:
- * Remove deprecated ``constant`` as function state modifier from documentation and tests (but still leave it as a valid feature).
- * Build System: Update internal dependency of jsoncpp to 1.8.4, which introduces more strictness and reduces memory usage.
+Language Features:
  * Code Generator: Use native shift instructions on target Constantinople.
```

Comment:
I'm not sure this is a language or compiler feature.

Reply:
I was also not 100% sure about this, but the general idea is that you can ignore compiler features more easily than language features and I think this is a rather important change.